specification/maps/data-plane/Search/preview/1.0/search.json

```diff
@@ -1551,6 +1754,7 @@
           {
             "name": "searchAddressBatchRequestBody",
             "in": "body",
+            "x-ms-client-flatten": true,
```

Comment:
This flattening has caused the `searchAddressBatchRequestBody` parameter to disappear in TS SDK. 
And the `batchItem` will be inside `options` and not a required parameter anymore.
![image](https://user-images.githubusercontent.com/18752396/136129170-47e48b3e-1e82-4c97-8586-4ab19a54a6f2.png)